projects/plugins/social/readme.txt

```diff
@@ -91,25 +91,16 @@ The easiest way is to use the Custom Message option in the publishing options bo
 4. Manage your Jetpack Social and other Jetpack plugins from My Jetpack.
 
 == Changelog ==
-### 1.4.0 - 2022-10-06
+### 1.5.0-beta - 2022-10-27
 #### Added
-- Add ContextualUpgradeTrigger to Jetpack Social admin page
-- Added check to not show the share metre if someone has a paid plan.
-- Added Jetpack social redirect urls.
-- Add pricing table to Jetpack Social
-- Adds ability to autotag, autorelease and autopublish releases
-- Enforce sharing limits in the Classic Editor
+- Display broken connections to user in editor
+- Integrate the ConnectionError react component to the Social plugin.
+- Reshare: Added the reshare UI to the block editor extension
 
 #### Changed
-- Changed the values on the pricing table, and fixed a redirect
-- Move share limits code to the Publicize package
-- Set version to 1.4.0-alpha
-- Social: Aligned Jetpack and Social to use the connection-test-results endpoint in the block editor
-- Updated package dependencies.
-- Updated style for Jetpack Logo icon shown in pre-publish panels for Jetpack and Jetpack Social plugins
-- Update Inspector Panel Jetpack icon color to #1E1E1E
-- Use Jetpack logo in Jetpack Social pre-publish screen for Publicize and Social Preview features
+- Updated package dependencies. [#25993, #26640, #26683, #26705, #26716, #26790, #26791, #26808, #26826, #26829, #26851, #27089]
```

Comment:
I don't think the package dependency updates need to be visible in the readme, but either way you ought to remove the PR numbers:

```suggestion
- Updated package dependencies.
```